### ISSUE TITLE
Check failed: status == CUBLAS_STATUS_SUCCESS (11 vs. 0)  CUBLAS_STATUS_MAPPING_ERROR - FCN

### ISSUE THREAD

**Issue by GaryKT** [BVLC/caffe#5399]
### Issue summary
Training FCN fails with Check failed: status == CUBLAS_STATUS_SUCCESS (11 vs. 0)  CUBLAS_STATUS_MAPPING_ERROR
Should there be a clearer error message? What causes this error?

I0313 17:15:14.385298 18356 solver.cpp:56] Solver scaffolding done.
[libprotobuf WARNING D:\build\caffe-builder\build_v140_x64\packages\protobuf\protobuf_download-prefix\src\protobuf_download\src\google\protobuf\io\coded_stream.cc:605] Reading dangerously large protocol message.  If the message turns out to be larger than 2147483647 bytes, parsing will be halted for security reasons.  To increase the limit (or to disable these warnings), see CodedInputStream::SetTotalBytesLimit() in google/protobuf/io/coded_stream.h.
[libprotobuf WARNING D:\build\caffe-builder\build_v140_x64\packages\protobuf\protobuf_download-prefix\src\protobuf_download\src\google\protobuf\io\coded_stream.cc:82] The total number of bytes read was 537963413
I0313 17:15:15.504062 18356 solver.cpp:219] Iteration 0 (-1.87257e-38 iter/s, 0.559091s/20 iters), loss = 2.26867e+06
I0313 17:15:15.504062 18356 solver.cpp:238]     Train net output #0: loss = 2.26867e+06 (* 1 = 2.26867e+06 loss)
I0313 17:15:15.505066 18356 sgd_solver.cpp:105] Iteration 0, lr = 1e-14
F0313 17:15:18.548606 18356 math_functions.cu:141] Check failed: status == CUBLAS_STATUS_SUCCESS (11 vs. 0)  CUBLAS_STATUS_MAPPING_ERROR
*** Check failure stack trace: ***

### Steps to reproduce

python backup_solve.py 0

Source:
import caffe
import surgery, score

import numpy as np
import os
import sys

try:
    import setproctitle
    setproctitle.setproctitle(os.path.basename(os.getcwd()))
except:
    pass

weights = 'fcn8s-heavy-pascal.caffemodel'

# init
caffe.set_device(int(sys.argv[1]))
caffe.set_mode_gpu()

solver = caffe.SGDSolver('solver.prototxt')
solver.net.copy_from(weights)

# surgeries
interp_layers = [k for k in solver.net.params.keys() if 'up' in k]
surgery.interp(solver.net, interp_layers)

solver.step(400)

# scoring
#val = np.loadtxt('../data/segvalid11.txt', dtype=str)

for _ in range(25):
    solver.step(400)
#    score.seg_tests(solver, False, val, layer='score')

solver.net.save('test_output.caffemodel')

### Your system configuration
Operating system: Windows
Compiler: Ninja / MSV14
CUDA version (if applicable): 8.0
CUDNN version (if applicable): 5.1
BLAS: ?
Python or MATLAB version (for pycaffe and matcaffe respectively): Python 3.5.2

**Comment by shelhamer** [comment]
Please ask usage questions on the mailing list. (There are threads for this error—it's usually a mismatch between number of outputs and number of labels.)

From https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md:

> _Please do not post usage, installation, or modeling questions, or other requests for help to Issues._
Use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) instead. This helps developers maintain a clear, uncluttered, and efficient view of the state of Caffe.

**Comment by ydas** [comment]
I got the same error in matlab and found out it was caused by missing the statement 'clear mex' at the beginning.
